### ISSUE TITLE
Line Component,  Shape=smooth, The inflection point is not on the coordinate point

### ISSUE THREAD

**Issue by hy-pillow** [antvis/F2#1509]
- [ ] I have searched the [issues](https://github.com/antvis/f2/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/youthful-albattani-cdb91w?file=/index.jsx)

### Steps to reproduce
无

| Environment | Info |
|---|---|
| f2 | 4.0.22 |
| System | - |
| Browser | - |

**Comment by zengyue** [comment]
Yes, smooth will make the curve look better, but it will lose accuracy. If you want to express accurately, it is not recommended to use smooth

**Comment by hy-pillow** [comment]
Can  ensure the xAxis of the inflection point~

![image](https://user-images.githubusercontent.com/6457155/172982349-e292382b-7bb8-47f1-a612-2914d6781137.png)

![image](https://user-images.githubusercontent.com/6457155/172982459-6dd6ebad-35d1-4aac-8918-782cfbf58b9c.png)

seems sometimes accurate, sometimes inaccurate

**Comment by zengyue** [comment]
really not sure, smooth will only ensure that the curve passes through the data points

**Comment by hy-pillow** [comment]
Thanks, but this may cause the curve to trace beyond the highest and lowest points of the data
